ADR/ADR036-add-solidqueue-to-forms-runner.md

```diff
@@ -0,0 +1,38 @@
+# ADR036: SolidQueue and ActiveJob for managing background processing
+
+Date: 2024-11-25
+
+## Status
+
+Approved
+
+## Context
+
+Forms will be using AWS SES to deliver emails to form processors with files attached to the emails. For processing of these emails, we looked at Sidekiq and Redis for managing a queue and to give us the ability to retry failed submissions - such as bounced emails. However with Redis, the details of a submission are ephemeral once a job is taken to be processed.
+Due to the way Sidekiq uses Redis in the free version is it possible for it to lose jobs if the process crashes. It is very important that we do not lose submissions, so this risk isn't acceptable to us.
```

Comment:
It might make sense to link out to [Sidekiq's reliability page](https://github.com/sidekiq/sidekiq/wiki/Reliability) here to make it easier for people to check whether it's still true in the future

Reply:
I'm not sure it's worthwhile doing this here...